mne/io/edf/edf.py

```diff
@@ -1541,6 +1584,12 @@ def read_raw_edf(
 
     Notes
     -----
+    :class:`mne.io.Raw` only stores signals with matching sampling frequencies.
+    Therefore, if mixed sampling frequency signals are requested, all signals
+    are upsampled to the highest loaded sampling frequency. In this case, using
+    preload=True is recommended, as otherwise, edge artifacts appear when
+    slices of the signal are requested.
```

Comment:
This should be a `%(edf_resamp_note)s` or so and go in `mne/utils/docs.py` to DRY the code

Reply:
done